x/emissions/keeper/inference_synthesis/network_regrets_test.go

```diff
@@ -860,93 +862,170 @@ func (s *InferenceSynthesisTestSuite) TestUpdateTopicInitialRegret() {
 	require.NotEqual(topic.InitialRegret, initialRegret)
 }
 
-// TestNotUpdateTopicInitialRegret tests that the topic's initial regret is not updated
-// when there are no experienced workers for the topic.
+// TestCalcSetNetworkRegretsWithFallbackRegrets tests that when there are no experienced workers
+// but enough total workers (>10), the system falls back to using all workers' regrets for
+// calculating the initial regret.
 //
 // Setup:
-// - Create a topic with an initial regret of 0
-// - Set up inferer and forecaster addresses
-// - Do not increment the inclusion counts for workers, leaving them as inexperienced
-// - Set regrets from the previous epoch
-// - Generate network losses for the current epoch
+// - Create a topic with a high alpha regret (0.5) so workers need 2 inclusions to be experienced
+// - Add 10 inferers and 2 forecasters with only 1 inclusion each (not experienced)
+// - Set initial regrets
+// - Create network losses value bundle with fixed values
+//
+// Test steps:
+// 1. Call GetCalcSetNetworkRegrets with the network losses
+// 2. Retrieve the updated topic
 //
 // Expected outcomes:
-//  1. The GetCalcSetNetworkRegrets function should execute without error
-//  2. The topic's initial regret should remain unchanged (0) after the function call
-//     because there are no experienced workers to trigger an update
+// 1. The function should execute without error
+// 2. Since we're using fallback regrets (no experienced workers but >10 total workers),
+// the initial regret should be the 25th percentile of all workers' regrets
+// without the offset calculation
+// 3. The new initial regret should match the expected value based on the fixed regrets
 //
-// This test ensures that the initial regret of a topic is not modified when
-// there are no experienced workers, maintaining the system's integrity by
-// preventing premature updates to the topic's regret baseline.
-func (s *InferenceSynthesisTestSuite) TestNotUpdateTopicInitialRegret() {
+// This test ensures that the fallback regret calculation works correctly when there
+// are no experienced workers but enough total workers to use the fallback mechanism.
```

Comment:
do we have a corresponding test for when the system is not at a cold start and there are >10 experienced workers?

Reply:
Yeah, in the same file, we have a couple of tests that cover the non-cold start part.

Reply:
what are those tests, I was struggling to figure them out

Reply:
`TestUpdateTopicInitialRegret`